kubernetes/samples/scripts/create-soa-domain/create-soa-domain-pv-pvc/README.md

```diff
@@ -0,0 +1,228 @@
+# Sample persistent volume and persistent volume claim
```

Comment:
Can the pv/c scripts be replaced with the common kubernetes/samples/scripts/create-weblogic-domain-pv-pvc scripts?

Reply:
Delete this directory create-database-pv-pvc.